src/v/cloud_storage/remote.cc

```diff
@@ -1269,7 +1270,15 @@ ss::future<upload_result> remote::delete_objects_sequentially(
                          });
                    })
             .handle_exception_type([ctxlog](const std::exception& ex) {
-                vlog(ctxlog.error, "Failed to delete keys: {}", ex.what());
+                if (ssx::is_shutdown_exception(std::make_exception_ptr(ex))) {
+                    vlog(
+                      ctxlog.debug,
+                      "Failed to delete keys during shutdown: {}",
+                      ex.what());
+
+                } else {
+                    vlog(ctxlog.error, "Failed to delete keys: {}", ex.what());
```

Comment:
this would be a good place for `vlogl`, eg

```
auto log_level = ...; 
vlogl(ctxlog, log_level, ...);
```

but the messages are different